src/parsers/poscar.js

```diff
@@ -63,8 +63,17 @@ export function atomsCount(poscarFileContent) {
     return atomsLine.map((x) => parseInt(x, 10)).reduce((a, b) => a + b);
 }
 
+export function materialNameFromFileContents(name, fileContent) {
```

Comment:
Docstring??

Reply:
Also - don't we already have XYZ too?

Reply:
Yes, we have XYZ. But that is in a separate branch which is currently not passing in jenkins. So i want to get this implemented with POSCAR first and then expand to xyz files. It should be a very straightforward update to make once poscar is implemented

Reply:
I added the XYZ function, but it is not being called at this time.